### PR TITLE
RSC-130 : Add in contributing.md

### PR DESCRIPTION
**Pull request by timlevett**
Took from https://github.com/sonatype-nexus-community/nexus-repository-conan/blob/master/.github/CONTRIBUTING.md and doctored up for our use case.

### CODE REVIEW

**Comment by timlevett** [comment]
@rpokorny 
>I thought the plan was that we weren't going to be particularly open to outside contributors

I wanted to follow what we do on others, which is just "if you are an outsider, talk with an expert on our Gitter. I highly doubt anyone will jump on it, but if they did happen to come across it, I wanted to give them a place to go.

File: .github/CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+Looking to dive in? FANTASTIC! Reach out to one of our experts and we can help. Hit us up on our Slack channel #react-components.
+
+## For Non-Sonatypers
+
+* Come hang out with us at our [gitter channel](https://gitter.im/sonatype/nexus-developers).
+* Sign the [Sonatype CLA](https://sonatypecla.herokuapp.com/sign-cla)
+
+## Are you submitting a pull request?
+
+* Use [our codestyle](https://github.com/sonatype/codestyle). If we get a PR that doesn't match it, there will be
```
[the remark this answers is not in the document]
Comment (timlevett) [reply .github/CONTRIBUTING.md]: Done in 22c8a60

**Comment by rpokorny** [comment]
We should probably move the PR etiquette section from the README to this doc

File: .github/CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+    the terms of the Eclipse Public License 2.0 which accompanies this
+    distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+
+-->
+
+# How to be a contributor to this project
+
+## Talk to a human first
+
+Looking to dive in? FANTASTIC! Reach out to one of our experts and we can help. Hit us up on our Slack channel #react-components.
```
Comment (rpokorny) [inline .github/CONTRIBUTING.md:14]: Should be create a "For Sonatypers" section and move the mentions of slack and JIRA to it?

**Comment by timlevett** [comment]
>We should probably move the PR etiquette section from the README to this doc

https://github.com/sonatype/sonatype-react-shared-components/pull/46/commits/e8d7e7c030dc4403908b4f212247957eeeb0031c

File: .github/CONTRIBUTING.md (hunk 2 above)
Comment (timlevett) [reply .github/CONTRIBUTING.md]: https://github.com/sonatype/sonatype-react-shared-components/pull/46/commits/58ad6690752611df2ed4799d4b5271a8f8e205c6

**Comment by timlevett** [comment]
Thanks for the help folks!

**Pull request merged**